nifi-docs/src/main/asciidoc/expression-language-guide.adoc

```diff
@@ -963,6 +963,36 @@ Expressions will provide the following results:
 	will return 0.
 
 
+[.function]
```

Comment:
Although it returns a String, I'm leaning more towards this being in the "Boolean Logic" sub-section. Thoughts?